// cat-me/cats.js
module.exports = {
  grumpy: '█▄░░░░░░░░░░░░░░░░░░░░░░░░▄▄███' + '\n' +
          '███▄░░░░░░░░░░░░░░░░░░░░▄██████' + '\n' +
          '█████▄░░░░░░░░░░░░░░░░░▄███████' + '\n' +
          '███████▄░░░░▄▄▄▄▄░░░░▄█████████' + '\n' +
          '█████████▄▀▀░░░░░▀▀▀▄██████████' + '\n' +
          '▀█████▀░░░░░░░░░░░░░░▀████████░' + '\n' +
          '░▀██▀░░░░░░░░░░░░░░░░░░░▀████▌░' + '\n' +
          '░░██░░░░░░░░░░░░░░░░░░░░░░███░░' + '\n' +
          '░░█▀░░░░░░░░░░░░░░░░░░░░░░░██░░' + '\n' +
          '░░█░░▄████▄░░░░░▄████▄░░░░░░█░░' + '\n' +
          '░░█░░█▐▄█▐█░░░░░█▐▄█▐█░░░░░░█▄░' + '\n' +
          '░░█░░██▄▄██░░░░░██▄▄██░░░░░░░█░' + '\n' +
          '░▐▌░░░░░░░░░░░░░░░░░░░░░░░░░░▐▌' + '\n' +
          '░▐▌░░░░░░░▀▄▄▄▄▀░░░░░░░░░░░░░▐▌' + '\n' +
          '░▐▌░░░░░░░░░▐▌░░░░░░░░░░░░░░░▐▌' + '\n' +
          '░▐▌░░░░░░░▄▀▀▀▀▄░░░░░░░░░░░░░▐▌' + '\n' +
          '░░█▄░░░░░▀░░░░░░▀░░░░░░░░░░░░█▌' + '\n' +
          '░░▐█▀▄▄░░░░░░░░░░░░░░░░░░▄▄▀▀░█' + '\n' +
          '░▐▌░░░░▀▀▄▄░░░░░░░░▄▄▄▄▀▀░░░░░█' + '\n' +
          '░█░░░░░░░░░▀▀▄▄▄▀▀▀░░░░░░░░░░░█' + '\n' +
          '▐▌░░░░░░░░░░░░░░░░░░░░░░░░░░░░█' + '\n' +
          '▐▌░░░░░░░░░░░░░░░░░░░░░░░░░░░░█',

  approaching: '   /\\     /\\' + '\n' +
               '  {  `---\'  }' + '\n' +
               '  {  O   O  }' + '\n' +
               '  ~~>  V  <~~' + '\n' +
               '   \\  \\|/  /' + '\n' +
               '    `-----\'__' + '\n' +
               '    /     \\  `^\\_' + '\n' +
               '   {       }\\ |\\_\\_   W' + '\n' +
               '   |  \\_/  |/ /  \\_\\_( )' + '\n' +
               '    \\__/  /(_E     \\__/' + '\n' +
               '      (  /' + '\n' +
               '       MM',

  tubby: '  /\\ ___ /\\' + '\n' +
         ' (  o   o  )' + '\n' +
         '  \\  >#<  /' + '\n' +
         '  /       \\' + '\n' +
         ' /         \\       ^' + '\n' +
         '|           |     //' + '\n' +
         ' \\         /    //' + '\n' +
         '  ///  ///   --',

  confused: '    /\\__/\\' + '\n' +
            '   /`    \'\\' + '\n' +
            ' === 0  0 ===' + '\n' +
            '   \\  --  /' + '\n' +
            '  /        \\' + '\n' +
            ' /          \\' + '\n' +
            '|            |' + '\n' +
            ' \\  ||  ||  /' + '\n' +
            '  \\_oo__oo_/#######o',

  playful: ' /\\     /\\' + '\n' +
           '{  `---\'  }' + '\n' +
           '{  O   O  }' + '\n' +
           '~~>  V  <~~' + '\n' +
           ' \\  \\|/  /' + '\n' +
           '  `-----\'____' + '\n' +
           '  /     \\    \\_' + '\n' +
           ' {       }\\  )_\\_   _' + '\n' +
           ' |  \\_/  |/ /  \\_\\_/ )' + '\n' +
           '  \\__/  /(_/     \\__/' + '\n' +
           '    (__/',

  thoughtful: '                   ;,_            ,' + '\n' +
              '                  _uP~"b          d"u,' + '\n' +
              '                 dP\'   "b       ,d"  "o' + '\n' +
              '                d"    , `b     d"\'    "b' + '\n' +
              '               l] [    " `l,  d"       lb' + '\n' +
              '               Ol ?     "  "b`"=uoqo,_  "l' + '\n' +
              '             ,dBb "b        "b,    `"~~TObup,_' + '\n' +
              '           ,d" (db.`"         ""     "tbc,_ `~"Yuu,_' + '\n' +
              '         .d" l`T\'  \'=                      ~     `""Yu,' + '\n' +
              '       ,dO` gP,                           `u,   b,_  "b7' + '\n' +
              '      d?\' ,d" l,                           `"b,_ `~b  "1' + '\n' +
              '    ,8i\' dl   `l                 ,ggQOV",dbgq,._"  `l  lb' + '\n' +
              '   .df\' (O,    "             ,ggQY"~  , @@@@@d"bd~  `b "1' + '\n' +
              '  .df\'   `"           -=@QgpOY""     (b  @@@@P db    `Lp"b,' + '\n' +
              ' .d(                  _               "ko "=d_,Q`  ,_  "  "b,' + '\n' +
              ' Ql         .         `"qo,._          "tQo,_`""bo ;tb,    `"b,' + '\n' +
              '(qQ         |L           ~"QQQgggc,_.,dObc,opooO  `"~~";.   __,7,' + '\n' +
              '`qp         t\\io,_           `~"TOOggQV""""        _,dg,_ =PIQHib.' + '\n' +
              ' `qp        `Q["tQQQo,_                          ,pl{QOP"\'   7AFR`' + '\n' +
              '   `         `tb  \'""tQQQg,_             p" "b   `       .;-.`Vl\'' + '\n' +
              '              "Yb      `"tQOOo,__    _,edb    ` .__   /`/\'|  |b;=;.__' + '\n' +
              '                            `"tQQQOOOOP""        `"\\QV;qQObob"`-._`\\_~~-._' + '\n' +
              '                                 """"    ._        /   | |oP"\\_   ~\\ ~\\_  ~\\' + '\n' +
              '                                         `~"\\ic,qggddOOP"|  |  ~\\   `\\  ~-._' + '\n' +
              '                                           ,qP`"""|"   | `\\ `;   `\\   `\\' + '\n' +
              '                                _        _,p"     |    |   `\\`;    |    |' + '\n' +
              '                                 "boo,._dP"       `\\_  `\\    `\\|   `\\   ;' + '\n' +
              '                                  `"7tY~\'            `\\  `\\    `|_   |' + '\n' +
              '                                                           `~\\  |',

  delighted: '                    .............                .""".             .""".' + '\n' +
             '            ..."""""             """""...       $   . ".         ." .   $' + '\n' +
             '        ..""        .   .   .   .   .    ..    $   $$$. ". ... ." .$$$   $' + '\n' +
             '      ."    . " . " . " . " . " . " . " .  "" ."  $$$"""  "   "  """$$$  ".' + '\n' +
             '    ."      . " . " . " . " . " . " . " .     $  "                    "   $' + '\n' +
             '   ."   . " . " . "           "   " . " . "  ."      ...          ...     ".' + '\n' +
             '  ."    . " . "    .."""""""""...     " . "  $     .$"              "$.    $' + '\n' +
             ' ."     . " . " .""     .   .    ""..   . " $ ".      .""$     .""$      ." $' + '\n' +
             '."    " . " .       . " . " . " .    $    " $ "      "  $$    "  $$       " $' + '\n' +
             '$     " . " . " . " . " . " . " . "   $     $             $$.$$             $' + '\n' +
             '$     " . " . " . " . " . " . " . " .  $  " $  " .        $$$$$        . "  $' + '\n' +
             '$     " . " . " . " . " . " . " . " .  $    $      "  ..   "$"   ..  "      $' + '\n' +
             '".    " . " . " . " . " . " . " . "   ."  "  $  . . . $  . .". .  $ . . .  $' + '\n' +
             ' $    " . " . " . " . " . " . " . "  ."   "            ".."   ".."' + '\n' +
             '  $     . " . " . " . " . " . "   .."   . " . "..    "             "    .."' + '\n' +
             '  ".      " . " . " . " . " .  .""    " . " .    """$...         ...$"""' + '\n' +
             '   ". "..     " . " . " . " .  "........  "    .....  ."""....."""' + '\n' +
             '     ". ."$".....                       $..."$"$"."   $".$"... `":....' + '\n' +
             '       "".."    $"$"$"$"""$........$"$"$"  ."."."  ...""      ."".    `"".' + '\n' +
             '           """.$.$." ."  ."."."    ."."." $.$.$"""".......  ". ". $ ". ". $' + '\n' +
             '                  """.$.$.$.$.....$.$.""""               ""..$..$."..$..$.',

  nyan: '  ☆    ☆  ☆      ' + '\n' +
        '┈┈┈┈ ╭━━━━━━╮  ☆   ' + '\n' +
        '┈☆ ┈┈┃╳╳╳▕╲▂▂╱▏  ' + '\n' +
        '┈┈☆ ┈┃╳╳╳▕▏▍▕▍▏  ' + '\n' +
        '┈┈ ╰━┫╳╳╳▕▏╰┻╯▏  ' + '\n' +
        '☆ ┈┈┈┃╳╳╳╳╲▂▂╱   ' + '\n' +
        '   ☆ ╰┳┳━━┳┳╯   ☆',

  resting: '                                |        |' + '\n' +
           '                                |\\      /|' + '\n' +
           '                                | \\____/ |' + '\n' +
           '                                |  /\\/\\  |' + '\n' +
           '                               .\'___  ___`.' + '\n' +
           '                              /  \\|/  \\|/  \\' + '\n' +
           '             _.--------------( ____ __ _____)' + '\n' +
           '          .-\' \\  -. | | | | | \\ ----\\/---- /' + '\n' +
           '        .\'\\  | | / \\` | | | |  `.  -\'`-  .\'' + '\n' +
           '       /`  ` ` \'/ / \\ | | | | \\  `------\'\\' + '\n' +
           '      /-  `-------.\' `-----.       -----. `---.' + '\n' +
           '     (  / | | | |  )/ | | | )/ | | | | | ) | | )' + '\n' +
           '      `._________.\'_____,,,/\\_______,,,,/_,,,,/'
}
